packages/web/src/services/http/http.service.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+import { api } from 'config';
+
+export const http = axios.create({
+  baseURL: api,
+})
+
+http.interceptors.request.use(
```

Comment:
Can we split it to separate files somehow?

Reply:
We could, but I don't really see a point to comlicate things just yet. As agreed, I'll show you the alternative way to do it for a more complex setup.